src/styles/components/_component-link.scss

```diff
@@ -0,0 +1,6 @@
+.component-link {
+  display: block;
+  outline: none;
+  width: 100%;
+  height: 100%;
+}
```

Comment:
After you rebase, I think is what the new `component-link` file should look like: 

````
.component-link {
  display: block;
  outline: none;
  width: 100%;
  height: 100%;
  background-color: var(--background-color);

  &:focus .component-card {
    position: relative;
    z-index: 1;
    outline: 2px solid var(--accent-color);
    outline-offset: 2px;
    transform: rotate(2deg);
  }
}
````

That makes sure that the background color is still picked up and the newer focus states are on.

Reply:
Let's try and find a time to huddle if you can, I have a few questions! :)